### PR TITLE
feat: Creating Link component

### PR DESCRIPTION
**Pull request by Delyc**
###  PR DESCRIPTION

- This PR creates arrow component together with link

### TASKS COMPLETED

- [x] Having the arrow rotating and animating when hovering on link text or the arrow

### HOW CAN THIS PR BE TESTED

```
git clone https://github.com/TheGymRwanda/unit8-maroon
git checkout ft/arrow-component
npm install
npm run dev

```

### BEFORE SUBMITTING YOUR PR, ENSURE IT HAS THE FOLLOWING

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard to understand areas
- [x] My changes generate no new warnings

### TICKET NUMBER

#O23: https://www.notion.so/apeunit/Arrow-Component-e2e766e98eb048dead7e2b8c8fc5057d

### CODE REVIEW

**Review by Nkaka23dev — COMMENTED** [review]
Cod looks good! The remaining is its preview to check if it matches with that of figma

File: pages/index.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
-import Tag from "../components/ui/Tag";
-
+import Link from "../components/UI/Link";
```
Comment (ishoborabyose) [inline pages/index.js]: It looks good, but you have to change The way you are calling The component , The UI should equal to Ui.

File: pages/index.js [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
 export default function Home() {
   return (
     <div className="">
```
Comment (ishoborabyose) [inline pages/index.js]: The className should not be empty

File: components/ui/Link.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      <>
+        <a
+         
+          className={` group flex  w-fit items-center gap-1.5 ${props.style}`}
+          href={props.link}
+        >
+          <p className={`font-Neufile text-2xl ${props.textStyle}`}>
+            {props.text}
+          </p>
+          <div className={`relative h-3.34 w-3.5 ${props.imageWrapperStyle}`}>
```
Comment (ishoborabyose) [inline components/ui/Link.jsx:11]: I can't see where you defined the height and width in config file, You should define height and width in the config file.

**Review by ishoborabyose — CHANGES_REQUESTED** [review]

**Review by haroldmud — APPROVED** [review]
LGTM ✔️

**Review by Nkaka23dev — APPROVED** [review]

**Review by ishoborabyose — APPROVED** [review]
# LGTM

**Pull request merged**